src/Cpp/1-getting-started/2-HelloFroge/Main.cpp

```diff
@@ -168,24 +158,11 @@ bool Initialize()
     constexpr char factoryName[] = "Factory1";
     g_Factory->SetPrivateData(WKPDID_D3DDebugObjectName, sizeof(factoryName), factoryName);
 
-    constexpr D3D_FEATURE_LEVEL d3dFeatureLevels[] =
-    {
-        D3D_FEATURE_LEVEL::D3D_FEATURE_LEVEL_11_0
-    };
+    constexpr D3D_FEATURE_LEVEL d3dFeatureLevels[] = {D3D_FEATURE_LEVEL::D3D_FEATURE_LEVEL_11_0};
     constexpr UINT deviceCreationFlags =
-        D3D11_CREATE_DEVICE_FLAG::D3D11_CREATE_DEVICE_BGRA_SUPPORT |
-        D3D11_CREATE_DEVICE_FLAG::D3D11_CREATE_DEVICE_DEBUG;
-    if (FAILED(D3D11CreateDevice(
-        nullptr,
-        D3D_DRIVER_TYPE::D3D_DRIVER_TYPE_HARDWARE,
-        nullptr,
-        deviceCreationFlags,
-        d3dFeatureLevels,
-        1,
-        D3D11_SDK_VERSION,
-        g_Device.ReleaseAndGetAddressOf(),
-        nullptr,
-        g_DeviceContext.ReleaseAndGetAddressOf())))
+        D3D11_CREATE_DEVICE_FLAG::D3D11_CREATE_DEVICE_BGRA_SUPPORT | D3D11_CREATE_DEVICE_FLAG::D3D11_CREATE_DEVICE_DEBUG;
+    if (FAILED(D3D11CreateDevice(nullptr, D3D_DRIVER_TYPE::D3D_DRIVER_TYPE_HARDWARE, nullptr, deviceCreationFlags, d3dFeatureLevels, 1,
```

Comment:
here it breaks apart again...

i mentioned earlier somewhere more than 1 param, should be on a new line (like on the left), hope that is possible

Reply:
```cpp
    if (FAILED(D3D11CreateDevice(
            nullptr, D3D_DRIVER_TYPE::D3D_DRIVER_TYPE_HARDWARE, nullptr, deviceCreationFlags, d3dFeatureLevels, 1, D3D11_SDK_VERSION,
            g_Device.ReleaseAndGetAddressOf(), nullptr, g_DeviceContext.ReleaseAndGetAddressOf())))
    {
```
is this better?

Reply:
That was the one like below I think, 1 value per row, if param count is >1